integration-tests/src/test/java/org/wildfly/prospero/it/cli/InstallTest.java

```diff
@@ -0,0 +1,46 @@
+package org.wildfly.prospero.it.cli;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.wildfly.prospero.cli.CliMain;
+import org.wildfly.prospero.cli.ReturnCodes;
+import org.wildfly.prospero.cli.commands.CliConstants;
+import org.wildfly.prospero.it.AcceptingConsole;
+import org.wildfly.prospero.test.MetadataTestUtils;
+import picocli.CommandLine;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class InstallTest {
```

Comment:
Just a note for future - I think for this sort of tests, we should use distributable generated by dist and run it as external process

Reply:
Good idea, I'm going to do that next.